RR.App.Tests/Controllers/HRIS/EmployeeAddressControllerIntegrationTests.cs

```diff
@@ -1,165 +1,137 @@
-                {
-                    services.AddScoped<IUnitOfWork, RR.UnitOfWork.UnitOfWork>();
-                    services.AddSingleton<IPolicyEvaluator, FakePolicyEvaluator>();
-                    services.AddScoped<IEmployeeAddressService, EmployeeAddressService>();
-                });
-            }).CreateClient();
-
-
-            using (var scope = _factory.Services.CreateScope())
-            {
-                var services = scope.ServiceProvider;
-                var context = services.GetRequiredService<DatabaseContext>();
-            }
-        }
 
-        [Fact]
-        public async Task GetAll_ReturnsOkResult()
-        {
-
-            var response = await _client.GetAsync("/employee-address");
+        var response = await _client.GetAsync("/employee-address");
 
-            response.EnsureSuccessStatusCode();
-            Assert.Equal(HttpStatusCode.OK, response.StatusCode);
-        }
-        [Fact]
-        public async Task SaveUpdateDeleteEmployeeAddress_ReturnsOkResult()
-        {
-            var addressDto = EmployeeAddressTestData.EmployeeAddressDtoNew;
-            var jsonContent = new StringContent(JsonConvert.SerializeObject(addressDto), Encoding.UTF8, "application/json");
-            
-            var response = await _client.PostAsync("/employee-address", jsonContent);
+        response.EnsureSuccessStatusCode();
+        Assert.Equal(HttpStatusCode.OK, response.StatusCode);
+    }
+    [Fact]
+    public async Task SaveUpdateDeleteEmployeeAddress_ReturnsOkResult()
+    {
+        var addressDto = EmployeeAddressTestData.EmployeeAddressDtoNew;
+        var jsonContent = new StringContent(JsonConvert.SerializeObject(addressDto), Encoding.UTF8, "application/json");
+        
+        var response = await _client.PostAsync("/employee-address", jsonContent);
 
-            response.EnsureSuccessStatusCode();
-            Assert.Equal(HttpStatusCode.OK, response.StatusCode);
+        response.EnsureSuccessStatusCode();
+        Assert.Equal(HttpStatusCode.OK, response.StatusCode);
 
-            var content = await response.Content.ReadAsStringAsync();
-            var jsonDoc = JsonDocument.Parse(content);
-            var addressId = jsonDoc.RootElement.GetProperty("id").GetInt32();
+        var content = await response.Content.ReadAsStringAsync();
+        var jsonDoc = JsonDocument.Parse(content);
+        var addressId = jsonDoc.RootElement.GetProperty("id").GetInt32();
 
-            var updatedDto = new EmployeeAddressDto
-            {
-                Id = addressId,
-                UnitNumber = "56",
-                ComplexName = "Complex72",
-                StreetNumber = "8",
-                SuburbOrDistrict = "Suburb/District",
-                City = "City",
-                Country = "Country",
-                Province = "Province",
-                PostalCode = "1620"
-            };
+        var updatedDto = new EmployeeAddressDto
```

Comment:
is there not test data that we could use for this and if there isnt can you please add this to test data